### DISCUSSION TITLE
Future of noisetorch

### DISCUSSION THREAD

**Discussion by lawl** [noisetorch/NoiseTorch#265] [General]
Some people seems to still want to maintain this. I don't know if i still can.

What of we move this project to an org, add some people, and then end me here.

Edit: end it for me, that sounded macabre.

**Comment by dlabaja** [comment]
It would be nice to continue this project someway, it's the only krisp alternative currently known to me and the stars speak for themselves.

**Comment by ZyanKLee** [comment]
Converting NoiseTorch into a community project where you are not the only one responsible and can have some backup from others might be a good choice anyway.

Although I understand that you need a break, I would be sad about you leaving the project altogether.

**Comment by ZyanKLee** [comment]
Somehow I may have been the one that was the loudest (and also created a few PRs). That seems to have gotten me in the pickles of now being co-owner of this project/organization. :-D
Thanks @lawl, for trusting me with this.

For how to continue, I am not sure yet. I'll be away for the weekend and may not respond regularly nor timely. But I will surely not get my head away from the topic of how I became the owner of a 6k stars project in merely hours.

I'm hoping @lawl will not leave us and we can somehow develop a plan with all the other contributors and followers on how to go on.

IMHO a good idea would be to structure the audit a little - perhaps someone can create a list with items to audit, which of them had been audited already by whom and which are still to be audited. Anyone up for the task?

**Reply by ZyanKLee** [reply]
Would anybody like to review my two action PRs? I'm kind of a newbie when it comes to GitHub actions (I only got experience with gitlab and Jenkins).

**Comment by Dadadah** [comment]
I'm very happy with the choice to move this to an org. I will gladly partake in this org!

**Comment by lawl** [comment]
Sorry guys, i have zero motivation to do anything computer related right now. I'll need to do something else, read a few books or something.

**Reply by lawl** [reply]
Have now left the noisetorch org.

**Reply by ZyanKLee** [reply]
Damn those bad guys. :-(
Have a good time and perhaps we meet again some day. Until then: farewell and enjoy the world.

**Comment by AXDOOMER** [comment]
Can we make sure every account that has write access to this repository has 2FA/MFA activated on their account?

**Reply by ZyanKLee** [reply]
I like that idea. In fact, since lawl left the organization this morning, that is definitely the case, as I'm currently the only one left with write access and my account is secured with 2fa.

:-/

**Reply by Technetium1** [reply]
@ZyanKLee since the org members are private, can you disclose how many people are in it right now?

**Reply by AXDOOMER** [reply]
and what can a member in the org do that someone that's not in it can't? So far, I've not seen any difference.

**Reply by Technetium1** [reply]
@AXDOOMER Normally it gives them permission to manage issues, discussions, trigger actions, and sometimes they can be required to sign off on pull requests depending on how it's set up. There's more info [here](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/roles-in-an-organization) and [here](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization).

**Reply by ZyanKLee** [reply]
At the moment, is sadly only me that has write access. I'll try to remedy that in the next few days.

Then I invited a few people that helped a lot during the last days to have read access:
* Axdoomer
* Dadadah
* TheDukeofErl

**Reply by ZyanKLee** [reply]
I really don't like it when projects have that high of a "bus factor".

**Comment by ZyanKLee** [comment]
Before releasing v0.12.0, I would only want to do one last thing: merge #299 

Every other issue we have, can in my opinion wait for the next patch or minor release. But bringing a new stable release out there to restore peoples workflow should imho not be pushed back anymore than necessary.

So please don't feel insulted, if I leave your PRs hanging out there. Most of them are still under active discussion and I don't feel we found a consent with them.
